src/applications/hca/config/migrations.js

```diff
@@ -158,7 +161,7 @@ export default [
   // required strings can not pass validation with only spaces
   ({ formData, metadata }) => {
     let newFormData = formData;
-    let newMetaData = metadata;
+    let newMetaData = metadata || {};
```

Comment:
I updated this because `newMetaData` was occasionally being to `undefined` from `metadata`. This was causing issues when used with our `platform/utilities/data/set` function.